components/layout_2020/flexbox/layout.rs

```diff
@@ -1033,15 +1069,16 @@ impl<'items> FlexLine<'items> {
     fn resolve_auto_main_margins(
         &self,
         remaining_free_space: Length,
-    ) -> impl Iterator<Item = (Length, Length)> + '_ {
+    ) -> (impl Iterator<Item = (Length, Length)> + '_, bool) {
+        let auto_margins_count = self
```

Comment:
If `remaining_free_space` is zero, treating `auto_margins_count` as zero should have no effect and would avoid iterating the items. But I guess it's fine either way.

Reply:
Right. According to spec, free space should only be distributed if the remaining free space is positive.